oclif/plugin-warn-if-update-available
# feat: use esm

experiment with ESM plugins in a non-ESM CLI.  Seems to work ok when I copy the js and pjson into `sf` locally

QA
1. yarn build so it sends files to /lib
2. manually copy the .js files from /lib and the package.json to your active CLI's node_modules/@oclif/plugin-plugin-warn-if-update-available

I was running this with DEBUG=* just to make sure the hook wasn't throwing warnings.
